### DISCUSSION TITLE
Platform specific views

### DISCUSSION THREAD

**Discussion by steven78-ed** [dotnet/maui#8671] [Q&A]
Hello,

Can someone explain me (or know an example), how i use different views for the different platforms. I.e. I have the same MainPage for all platforms, but i need  different detail pages for Android and Windows.

Best Regards
Steven

**Comment by xtuzy** [comment]
https://docs.microsoft.com/en-us/dotnet/maui/xaml/markup-extensions/consume#onplatform-markup-extension or use `if else` in code
